src/package_test.ts

```diff
@@ -7,4 +7,13 @@ describe('package', () => {
         const v2 = require('../package-lock.json').version;
         expect(v1).to.equal(v2);
     });
+
+    it('package-lock should only reference npm', () => {
```

Comment:
```suggestion
    it('package-lock should only reference official public npm registry', () => {
```

lets be more exact and specific here too.